frontend/src/component/pages/admin/changelog/ManageChangeLogs.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { Section } from '../../../ui/Section';
+import { Button } from '../../../ui/Button';
+import { Store } from 'redux';
+import { IAppState } from '../../../../state/reducers';
+import { ChangeLogService } from '../../../../service/ChangeLog.service';
+import { Modal } from '../../../ui/Modal';
+import './ManageChangeLogs.scss';
+import { CreateChangeSection } from './CreateChangeSection';
+import { Icon, IconID } from '../../../ui/Icon';
+
+interface IProps {
+  changeLogService: ChangeLogService;
+  notifyToast: (message: string, duration?: number) => void;
+  store: Store<IAppState>;
+  onAuthenticationFailed: () => void;
+}
+
+export class ManageChangeLogs extends Component<IProps> {
+  private createChangeModalRef = React.createRef<Modal>();
+
+  render() {
+    return (
+      <div className="manage-change-logs">
```

Comment:
```suggestion
      <div className="ChangeLogTab">
```
It seems that this is React's convention.